contrib/pkg/v1migration/restoreownerrefs.go

```diff
@@ -164,3 +171,12 @@ func isNamespaceScoped(kind string) bool {
 		return true
 	}
 }
+
+func isRefFromPVCToCD(ref ownerRef) bool {
+	return ref.Resource == "persistentvolumeclaims" && ref.Group == "" &&
+		ref.Ref.Kind == "ClusterDeployment" && ref.Group == hivev1alpha1.SchemeGroupVersion.String()
```

Comment:
Requires ref.Group == "" && ref.Group == hivev1alpha1, looks like a typo here, guessing the latter one should be ref.Ref.Group == hivev1alpha1

Reply:
Yes. Good catch. Looking through the owner-refs.yaml file, I don't see any refs from PVCs captured. So I need to look into why that was missed.

Changed to `ref.Ref.APIVersion == hivev1alpha1.SchemeGroupVersion.String()`.

Also, in `isRefFromDNSEndpoint`, the check for the group was wrong, since the group does not contain the version. I changed that to `ref.Group == hivev1alpha1.GroupName`.